### PR TITLE
feat(core/webidl): point the failing IDL block

### PR DESCRIPTION
**Pull request by saschanaz**
With some error style change:

Before:

image

After:

image

### CODE REVIEW

File: src/core/webidl.js [hunk 1]
```diff
@@ -284,12 +283,13 @@ function renderWebIDL(idlElement) {
...
-      `Failed to parse WebIDL: ${e.message}.
-      <details>
-      <pre>${idlElement.textContent}\n ${e}</pre>
-      </details>`
+    showInlineError(
+      idlElement,
+      `Failed to parse WebIDL: ${e.bareMessage}.`,
+      e.bareMessage,
+      {
+        details: `<pre>${e.context}</pre>`,
```
Comment (sidvishnoi) [inline src/core/webidl.js]: Nit: Can we do this instead?
``` js
{ details: `<pre>${e.context}</pre>` }
```

**Comment by sidvishnoi** [comment]
Why is there that much empty space after the details are open in second screenshot though?

File: src/core/webidl.js (hunk 1 above)
Comment (saschanaz) [reply src/core/webidl.js]: ESLint makes me do the longer way 🤔

Comment (saschanaz) [reply src/core/webidl.js]: Oh, it seems somehow it allows both ways. I'll apply your suggestion.

**Comment by saschanaz** [comment]
>Why is there that much empty space after the details are open in second screenshot though?

Oh, an empty `<p>` is there, hmm...

**Comment by saschanaz** [comment]
Adding proper newlines removes that empty paragraph. Markdown is tricky 🤣

**Pull request merged**